app/api/activitylog/activitylogMiddleware.js

```diff
@@ -38,13 +38,14 @@ export default (req, _res, next) => {
   if (mustBeLogged(baseurl, method, body)) {
     const time = Date.now();
     const expireAt = date.addYearsToCurrentDate(1);
-    if (body.password) body.password = '*****';
+    const bodyLog = { ...body };
```

Comment:
@RafaPolit if body is undefined then `{ ...body }` resolves to an empty object `{}`, would it be okay to leave it that way instead of introducing a branch? @konzz

Reply:
Yes, I think this is good enough.

Reply:
This is ready for QA @RafaPolit